cli/flags.rs

```diff
@@ -918,6 +923,13 @@ Ignore linting a file by adding an ignore comment at the top of the file:
         .long("rules")
         .help("List available rules"),
     )
+    .arg(
+      Arg::with_name("rule")
+        .long("rule")
+        .takes_value(true)
+        .multiple(true)
+        .help("Display docs of the specified rules"),
```

Comment:
Nit on the help text, maybe something like:
```suggestion
        .help("Display help for the specified rule"),
```

Reply:
To me "help" sounds like something that tells us how to deal with diagnostics. It's partially true but there might be a more appropriate word like... "description"?